slurm_make_example/README.md

```diff
@@ -0,0 +1,61 @@
+# `slurm_makefile_example`
+
+This example is for the case where a makefile guides the overall data analysis, 
+with one step that we want to conduct on a cluster. The non-cluster step in this
+very simple example is to create the jobs.csv file. The cluster step is to run 
+one job for each row of jobs.csv. Each job creates an output file.
+
+In addition, we assume that the cluster process is unreliable: it can't be 
```

Comment:
Ha, interesting. So FYI, one of the things I like about SLURM/YETI is the consistency of job completion (unless, as you mention, there is a flaky external dependency).